### PR TITLE
Downgrade to Unity2019

### PR DESCRIPTION
**Pull request by dstrawberrygirl**
If you can possibly ignore the 99k additions on this PR, what's actually changed here is the version of the editor and a few tweaks to the following prefabs:

- InformationPanel - split from the NewCanvasUI prefab, this is just the bit on the left that shows information
- InteractionMenu - split from the NewCanvasUI prefab, this is the bit on the right that is interactable. This includes the Selection panel with all the cities and constellations. Note that these are now pre-populated in the prefab because it'll make it soo much easier when it comes to VR positioning!
- SimManager so it can reference the main UI and the newly-separated InfoPanel prefab and instantiate both at launch

This main change is so that VR and mixed reality versions can then link the two prefabs to each controller.

While I had hoped to get the MRTK in here and leave it dormant on this branch, given the extent of the changes required to get MRTK working I subsequently removed it. If anything remains in here from that process it should be considered harmless :)

### CODE REVIEW

File: Assets/Scripts/UI/MenuController.cs [hunk 1]
```diff
@@ -32,24 +32,38 @@ private DataController dataController
...
         }
         else showHideMenuToggle.SendMessage("SetText", _menuShowIcon);
     }
+    public void ToggleInfoPanel()
+    {
+        showInfoPanel = !showInfoPanel;
+        informationPanelObject.SetActive(showInfoPanel);
+        if (showInfoPanel && showHideInfoPanelToggle != null)
+        {
+            // use SendMessage because text component on canvas ui vs world ui is different
```
Comment (knowuh) [inline Assets/Scripts/UI/MenuController.cs:62]: Wow, you will have to explain this to me sometime.  For now I am going to take you at your word :)

File: Assets/Scripts/UI/SelectionPanelController.cs [hunk 2]
```diff
@@ -7,44 +7,43 @@
...
         {
-            Debug.Log("found city list");
-            _cityList.InitCityNames(DataManager.Instance.CityNames, SimulationManager.Instance.CurrentLocationName);
+            StartCoroutine(cityList.InitCityNames(DataManager.Instance.CityNames, SimulationManager.Instance.CurrentLocationName));
         }
 
-        if (_cityDropdown)
+        if (cityDropdown)
         {
             Debug.Log("found city drop down");
```
Comment (knowuh) [inline Assets/Scripts/UI/SelectionPanelController.cs:33]: We can probably remove this debug statement soon.

**Review by knowuh — APPROVED** [review]
Looks good to me.  Thanks!

File: Assets/Scripts/UI/SelectionPanelController.cs (hunk 2 above)
Comment (dstrawberrygirl) [reply Assets/Scripts/UI/SelectionPanelController.cs:33]: Looking forward to the Great Purge, where we remove all these dead UI pieces :)

**Pull request merged**